AnyText/Tests/AnyTextExtension/srv/AnyText.anytext

```diff
@@ -0,0 +1,142 @@
+﻿grammar AnyText ( anytext )
```

Comment:
der Ordner srv hätte eigentlich nicht eingecheckt werden sollen, kannst du den bitte rauslöschen und in die .gitignore setzen?